services/connector/commands/personal_sign.go

```diff
@@ -29,18 +29,17 @@ func (r *RPCRequest) getPersonalSignParams() (*PersonalSignParams, error) {
 		return nil, ErrEmptyRPCParams
 	}
 
-	paramMap, ok := r.Params[0].(map[string]interface{})
-	if !ok {
+	if len(r.Params) < 2 {
 		return nil, ErrInvalidParamsStructure
 	}
 
-	// Extract the Challenge and Address fields from paramMap
-	challenge, ok := paramMap["challenge"].(string)
+	// Extract the Challenge and Address fields from paramsArray
+	challenge, ok := r.Params[0].(string)
```

Comment:
Is it possible to have a challenge at index 1 and address at index 0 or it's deterministically set like that?